mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb.inc
+
+if (`SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.ENGINES WHERE engine = 'rocksdb' AND support IN ('DEFAULT')`)
+{
+  --skip Test requires default engine RocksDB
+}
+
+# MYR-14
+#if (`select count(*) = 0 from information_schema.tables where engine='rocksdb' and table_name='slave_gtid_info'`)
```

Comment:
Is this check required? We'd support a mixed InnoDB + RocksDB setup, so probably we'd use the InnoDB crash-safe slave?